portfolio/src/main/webapp/index.html

```diff
@@ -1,113 +1,129 @@
 <!DOCTYPE html>
 <html>
-
   <head>
-
     <meta charset="UTF-8">
-
     <title>Shivani Ghanta</title>
-
     <!-- CSS -->
     <link rel="stylesheet" href="style.css">
     <link rel="stylesheet" href="bootstrap.css">
-
-    <!-- Fonts -->
-    <link href='https://fonts.googleapis.com/css?family=Aclonica:400,300,300italic,400italic,600,600italic,700,700italic,800,800italic' rel='stylesheet' type='text/css'>
-    <link href='https://fonts.googleapis.com/css?family=Adamina:400,100,200,300,500,600,700,800,900' rel='stylesheet' type='text/css'>
-
     <script src="script.js"></script>
-
   </head>
   <body>
-
-      <!-- Navigation Bar -->
-      <nav class="navbar navbar-default top">
-          <div class="container text-center">
-              <div class="navbar-header top">
-                  <div data-toggle="collapse" data-target="#myNavbar">
-                      <span class="sr-only">Toggle navigation</span>
-                      <span class="icon-bar"></span>
-                      <span class="icon-bar"></span>
-                      <span class="icon-bar"></span> 
-                  </div>
-              </div>
-
-    <div id="navbar" class="navbar-collapse">
-        <ul class="nav navbar-nav">
+    <!-- Navigation Bar -->
+    <nav class="navbar navbar-default top">
+      <div class="container text-center">
+        <div class="navbar-header top">
+          <div data-toggle="collapse" data-target="#myNavbar">
+          </div>
+        </div>
+        <div id="navbar" class="navbar-collapse">
+          <ul class="nav navbar-nav">
             <li><a href="index.html">Home</a></li>
-            <li><a href="resume.pdf">Resume</a></li>
+            <li><a href="documents/resume.pdf">Resume</a></li>
             <li><a href="mailto:shivani.ghanta@gmail.com">Email Me</a></li>
-
-        </ul>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <!-- Title Content & Bio -->
+    <div class="bitmoji-top">
+      <div class="container">
+        <div class="row">
+          <div class="about">
+            <div class="bio">
+              <h1>Shivani Ghanta</h1>
+              <h2>Aspiring Software Engineer</h2>
+              <hr class="top-bio">
+              <br>
```

Comment:
nit: typically, you'd want to use CSS to set the spacing below the header rather than using `<br />`